tools/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualAppInfo.scala

```diff
@@ -217,6 +230,7 @@ class QualAppInfo(
       }.mkString(";")
       val writeFormat = writeFormatNotSupported(writeDataFormat)
       val (allComplexTypes, nestedComplexTypes) = reportComplexTypes
+      val problems = getAllPotentialProblems(getPotentialProblems, nestedComplexTypes)
```

Comment:
it seems odd to have 2 getPotentialProblems functions, if they are specific rename the functions.  I would have expected this just to go into getPotentialProblems and I kind of would expect from the name findPotentialIssues would find these.  I'm fine if that doesn't make sense but we need to document better. Also findPotentialIssues is in the base which would let us use in profiling tool easily, this is not. It doesn't need to be at this point but just want code to be easy to follow.

Reply:
Updated the function name. Please let me know if the new name is okay.